lte/protos/subscriberdb.proto

```diff
@@ -271,3 +273,23 @@ message ListSubscribersResponse {
   // Deprecated
   reserved 3, 4;
 }
+
+message SuciProfile {
+  enum ECIESProtectionScheme {
+    ProfileA = 0;
+    ProfileB = 1;
+  }
+  // HPLMN Public Key Identifier
+  uint32 home_network_public_key_identifier = 1;
+  // HPLMN Public Key
+  bytes home_network_public_key = 2;
+  // HPLMN Private Key
```

Comment:
Nit: prefix the comments with the identifier name, e.g. `home_network_public_key is ...`

Reply:
done